### DISCUSSION TITLE
Failed tx on PasswordStore

### DISCUSSION THREAD

**Discussion by sunnyStefi** [Cyfrin/security-and-auditing-full-course-s23#114] [Q&A]
Hi! I saw [here](https://sepolia.etherscan.io/tx/0x5f0a6094d041fb59a35763f367e181c5760adbf4ebb8c0cb7d41809ec48dd7d4) that someone sent a failed tx to patrick's passwordStore. Any idea how he did it?
Thank you!
Stefi

**Comment by cromewar** [comment]
Hello @sunnyStefi 
The transaction reverted because it didn't use a valid input parameter:

the input was:

```
\m

 	hello pat
```

So, that's why it failed.
